### ISSUE TITLE
Connection info click area is on the urlbarIcon, should be on urlbarIconContainer

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#7888]
## Test plan
See https://github.com/brave/browser-laptop/pull/13147

## Original issue description
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
The click area is pretty small limited only to the urlbaricon, even a little off causes the click to be not captured. The click should be on the urlbarIconContainer which increases the hit area 

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version (revision SHA):
Brave	0.14.0
rev	ffa6f8f

- Steps to reproduce:
    1. Open any http/https site
    2. Click on the padlock icon the connection info is shown 
    3. Move cursor sightly off the padlock but visually the pointer still looks on the padlock, click not recognized and connection info is not shown 

  **All other browser has the large click area to show the connection info window/modal**
    
- Actual result:
Connection info click area is on the urlbarIcon, should be on urlbarIconContainer

- Expected result:
Connection info not shown if the click is off by a marginal amount

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
What it is now
![image](https://cloud.githubusercontent.com/assets/17010094/24321566/402ec562-1147-11e7-8153-c8e001507c7b.png)
What it should be 
![image](https://cloud.githubusercontent.com/assets/17010094/24321570/527c3e98-1147-11e7-8c94-0feff2401667.png)

- Any related issues: #6704
cc: @bradleyrichter @bsclifton

**Comment by luixxiul** [comment]
The current clickable are is the same as Chrome fyi

**Comment by srirambv** [comment]
No its not the same. Its completely different from Chrome
![7888](https://cloud.githubusercontent.com/assets/17010094/24324353/05af5286-117d-11e7-8982-52198a875712.gif)

**Comment by luixxiul** [comment]
sorry I was on Ubuntu

**Comment by luixxiul** [comment]
Still I believe that the clickable area is not only the lock icon but also the whole "GitHub, Inc. [US]" rectangle on your GIF animation.

![icon](https://cloud.githubusercontent.com/assets/3362943/24324605/a4a2e340-11cc-11e7-8e09-941d15e62a88.gif)

**Comment by srirambv** [comment]
Yes. Since we don't have any text next to the padlock it makes sense to have the entire urlbarIconContainer clickable. Users might be used to click on the text next to padlock and would expect the similar behavior. Just makes it easy to view.

**Comment by bsclifton** [comment]
this is a super important one to fix- do you think you could check this out @luixxiul?

Moving to 0.14.3 for now

**Comment by luixxiul** [comment]
Fine, I will add another container inside `urlbarIconContainer` to wrap it.

I don't think it is a good idea to make all of `urlbarIconContainer` clickable, unless we add the border like bookmark button and add fund button. This should be another task (if we will display some info like Chrome does).

**Comment by bsclifton** [comment]
@luixxiul I think that's a great point- @bradleyrichter would we be interested in adding a border?

**Comment by bradleyrichter** [comment]
a border? I like Chrome's "border". Invisible.  ;   )

**Comment by MargarytaChepiga** [comment]
Hey! Is there any help needed on this issue?

**Comment by srirambv** [comment]
@MargarytaChepiga  please open up a PR with the fix

**Comment by Liunkae** [comment]
The extended validation certificate info text should also be clickable (#13011).

**Comment by MargarytaChepiga** [comment]
@srirambv Working on it!

**Comment by bsclifton** [comment]
Awesome! thanks, @MargarytaChepiga 😄 I assigned the issue to myself to reserve it for you- let us know if you need any help 😄

**Comment by MargarytaChepiga** [comment]
@bsclifton Great! Thank you!

**Comment by Liunkae** [comment]
@MargarytaChepiga
The onClick function in urlBarIcon.js contains the code that shows the connection info.
https://github.com/brave/browser-laptop/blob/cd99dc66d1bf13153b5bfae7efdbd6e0d6708441/app/renderer/components/navigation/urlBarIcon.js#L124-L130

The onClick function is bound to the component here:
https://github.com/brave/browser-laptop/blob/cd99dc66d1bf13153b5bfae7efdbd6e0d6708441/app/renderer/components/navigation/urlBarIcon.js#L33

And the onClick function is set to the onClick attribute here:
https://github.com/brave/browser-laptop/blob/cd99dc66d1bf13153b5bfae7efdbd6e0d6708441/app/renderer/components/navigation/urlBarIcon.js#L173

The onClick function in urlBarIcon.js needs to be moved over to urlBar.js. Then, the onClick attribute of the urlbarIconContainer (on both lines 500 and 506) can be set to the new function.

https://github.com/brave/browser-laptop/blob/cd99dc66d1bf13153b5bfae7efdbd6e0d6708441/app/renderer/components/navigation/urlBar.js#L498-L510

**Comment by MargarytaChepiga** [comment]
@LiunkaeThat helps a lot! Thank you so much!
